libweb3jsonrpc/Eth.cpp

```diff
@@ -772,4 +781,16 @@ string dev::rpc::exceptionToErrorMessage()
 		ret = "Invalid RPC parameters.";
 	}
 	return ret;
-}
\ No newline at end of file
+}
+
+Ethash& Eth::getEthash()
+{
+    try
+    {
+        return asEthash(*client()->sealEngine());
+    }
+    catch (InvalidSealEngine&)
+    {
+        throw JsonRpcException("Seal Engine is not Ethash");
```

Comment:
Now this rethrowing doesn't make much sense, as all calling functions have either `catch(...)` or `catch(std::exception)`.
So I suggest to remove try/catch from this function